pyiron_atomistics/lammps/potentials.py

```diff
@@ -0,0 +1,414 @@
+
+    @property
+    def citations(self) -> str:
+        """Citations to be included"""
+        return "".join(np.unique([c for c in self.df.citations if len(c) > 0]))
+
+    @property
+    def is_scaled(self) -> bool:
+        """Scaling in pair_style hybrid/scaled and hybrid/overlay (whih is scale=1)"""
+        return "scale" in self.df
+
+    @property
+    def pair_style(self) -> str:
+        """LAMMPS pair_style"""
+        if len(set(self.df.pair_style)) == 1:
+            pair_style = "pair_style " + list(set(self.df.pair_style))[0]
+            if np.max(self.df.cutoff) > 0:
+                pair_style += f" {np.max(self.df.cutoff)}"
+            return pair_style + "\n"
+        elif "scale" not in self.df:
+            pair_style = "pair_style hybrid"
+        elif all(self.df.scale == 1):
+            pair_style = "pair_style hybrid/overlay"
+        else:
+            pair_style = "pair_style hybrid/scaled"
+        for ii, s in enumerate(self.df[["pair_style", "cutoff"]].values):
+            if pair_style.startswith("pair_style hybrid/scaled"):
+                pair_style += f" {self.df.iloc[ii].scale}"
+            pair_style += f" {s[0]}"
+            if s[1] > 0:
+                pair_style += f" {s[1]}"
+        return pair_style + "\n"
+
+    @property
+    def pair_coeff(self) -> list:
+        """LAMMPS pair_coeff"""
+
+        class PairCoeff:
+            def __init__(
+                self,
+                is_hybrid,
+                pair_style,
+                interacting_species,
+                pair_coeff,
+                species,
+                preset_species,
+            ):
+                self.is_hybrid = is_hybrid
+                self._interacting_species = interacting_species
+                self._pair_coeff = pair_coeff
+                self._species = species
+                self._preset_species = preset_species
+                self._pair_style = pair_style
+
+            @property
+            def counter(self):
+                """
+                Enumeration of potentials if a potential is used multiple
+                times in hybrid (which is a requirement from LAMMPS)
+                """
+                key, count = np.unique(self._pair_style, return_counts=True)
+                counter = {kk: 1 for kk in key[count > 1]}
+                results = []
+                for coeff in self._pair_style:
+                    if coeff in counter and self.is_hybrid:
+                        results.append(str(counter[coeff]))
+                        counter[coeff] += 1
+                    else:
+                        results.append("")
+                return results
+
+            @property
+            def pair_style(self):
+                """pair_style to be output only in hybrid"""
+                if self.is_hybrid:
+                    return self._pair_style
+                else:
+                    return len(self._pair_style) * [""]
```

Comment:
```suggestion
                return self._pair_style if self.is_hybrid else len(self._pair_style) * [""]
```

If this breaks line length limits just leave the explicit if/else clause you have; it's just hard to tell in the github editor

Reply:
Yeah it does.